static/app/utils/replays/replayReader.tsx

```diff
@@ -151,11 +152,14 @@ export default class ReplayReader {
 
   getMemorySpans = memoize(() => this.sortedSpans.filter(isMemorySpan));
 
-  isNetworkDetailsSetup = memoize(() =>
-    this.getNetworkSpans().some(
-      span =>
-        Object.keys(span.data.request?.headers || {}).length ||
-        Object.keys(span.data.response?.headers || {}).length
-    )
-  );
+  sdkConfig = memoize(() => {
+    const found = this.rrwebEvents.find(
+      event => event.type === 5 && event.data.tag === 'options'
+    ) as undefined | RecordingOptions;
+    return found?.data?.payload;
+  });
+
+  isNetworkDetailsSetup = memoize(() => {
+    return this.sdkConfig()?.networkDetailHasUrls;
+  });
```

Comment:
Should we fallback to old method in order to support users w/ SDK < 7.51.1?

Reply:
Thats a good idea, and we won't really need to bump the min version anymore. Done, and closed the related docs pr as not needed.